docs/README-AWS.md

```diff
@@ -626,9 +281,9 @@ Custom init arguments for the given forwarding target output
   * `args.username`: Username of the elasticsearch instance to connect to. Mandatory in case `args.api_key` is not provided. Will be ignored if `args.api_key` is defined as well.
   * `args.password` Password of the elasticsearch instance to connect to. Mandatory in case `args.api_key` is not provided. Will be ignored if `args.api_key` is defined as well.
   * `args.api_key`:  Api key of elasticsearch endpoint in the format **base64encode(api_key_id:api_key_secret)**. Mandatory in case `args.username`  and `args.password ` are not provided. Will take precedence over `args.username`/`args.password` if both are defined.
-  * `args.es_index_or_datastream_name`: Name of the index or data stream where to forward the logs to. Lambda supports automatic routing of various AWS service logs to the corresponding data streams for further processing and storage in the Elasticsearch cluster. It supports automatic routing of `aws.cloudtrail`, `aws.cloudwatch_logs`, `aws.elb_logs`, `aws.firewall_logs`, `aws.vpcflow`, and `aws.waf` logs. For other log types, if using data stream, the users can optionally set its value in the configuration file according to the naming convention for data streams and available integrations. If the `es_index_or_datastream_name` is not specified and it cannot be matched with any of the above AWS services then the value will be set to "logs-generic-default".
+  * `args.datastream`: Name of data stream or the index where to forward the logs to. Lambda supports automatic routing of various AWS service logs to the corresponding data streams for further processing and storage in the Elasticsearch cluster. It supports automatic routing of `aws.cloudtrail`, `aws.cloudwatch_logs`, `aws.elb_logs`, `aws.firewall_logs`, `aws.vpcflow`, and `aws.waf` logs. For other log types, if using data stream, the users can optionally set its value in the configuration file according to the naming convention for data streams and available integrations. If the `datastream` is not specified and it cannot be matched with any of the above AWS services then the value will be set to "logs-generic-default". Before **v0.30.0** this param was named `es_index_or_datastream_name`, that's now deprecated. It can still be used until the release of **v1.0.0**, when it will be finally removed.
```

Comment:
@ravikesarwani 
```
Before **v0.30.0** this param was named `es_index_or_datastream_name`, that's now deprecated. It can still be used until the release of **v1.0.0**, when it will be finally removed.
```

please, let me know if you prefer a different wording for the deprecation alert

Reply:
In version **v0.29.0** and before this configuration parameter was named `es_index_or_datastream_name`. Rename the configuration parameter to `es_datastream_name` in your configuration yaml file on the S3 bucket, to continue using it in the future version. The older name `es_index_or_datastream_name` is deprecated as of version **v0.30.0** and related backward compatibility code will be removed in version **v1.0.0**.